kotlin/services/stepfunctions/src/main/kotlin/com/kotlin/stepfunctions/HelloStepFunctions.kt

```diff
@@ -1,33 +1,32 @@
-// snippet-sourcedescription:[ListStateMachines.kt demonstrates how to list existing state machines for AWS Step Functions.]
+// snippet-sourcedescription:[HelloStepFunctions.kt demonstrates how to list existing state machines for AWS Step Functions.]
 // snippet-keyword:[AWS SDK for Kotlin]
 // snippet-service:[AWS Step Functions]
+
 /*
    Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
    SPDX-License-Identifier: Apache-2.0
 */
-
 package com.kotlin.stepfunctions
 
 // snippet-start:[stepfunctions.kotlin.list_machines.import]

```

Comment:
Is the hello service example supposed to include the imports? I see the snippet tags here but in the test build it is just the code. Here's the info for the hello service example: 

> Hello Step Functions

> Initialize the client and call ListStateMachines with maxResults=10. This may be an empty list if no state machines exist. Print the name and stateMachineArn of each state machine returned.

> General info for Hello Service example snippets:
This section of the MVP should be a streamlined, simple example with enough detail to be as close to “copy/paste” runnable as possible. This example may include namespaces and other setup in order to focus on getting the user up and running with the new service.

Reply:
I have modified the start of the tag so imports will be included.